e2e/testdata/hydration/compiled-json/helm-components/wordpress/secret_my-wordpress.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
What are these new files for?

Reply:
The helm chart changed, and these files are used by another test to verify the contents of the hydrated helm chart.